src/common/utils/time-utils.test.ts

```diff
@@ -77,11 +77,16 @@ describe('utc time formatting', () => {
 describe('date string parsing', () => {
   test('ISO format to date object', () => {
     expect(parseDateString('2020-03-01T00:00:00.000')).toEqual(
-      new Date(2020, 2, 1),
+      new Date(2020, 1, 29, 16),
```

Comment:
I was expecting this would break for some timezones, but I see we are overriding the `yarn jest` command to use `America/Los_Angeles` timezone, so this works as expected as long as you run the tests via that command...  But it'd be nice if our tests were not timezone-dependent for the most part.  Could we do `new Date('2020-03-01T00:00:00.000Z')` here instead?

Reply:
Yea, @pnavarrc and I decided to implement `America/Los_Angeles` timezone when we run tests to ensure tests are consistent timezone wise. I'm not sure exactly how our tests would interpret timezone if we do not specify a timezone at all (dependent on server?). I can definitely adjust the date to include `Z` though!

Reply:
Yeah, to be clear, I mean that instead of `new Date(2020, 1, 29, 16)` you could do `new Date('2020-03-01T00:00:00.000Z')`.  Since we're explicitly specifying the timezone (via `Z`), you don't need to know that the tests run in PDT which is +8 from UTC or whatever it is. 😁